codegenerator/cli/templates/static/codegen/src/eventFetching/hyperfuel/HyperFuel.res

```diff
@@ -261,10 +270,11 @@ module BlockData = {
     // If the block is not found, retry the query. This can occur since replicas of hypersync might not hack caught up yet
     if res.nextBlock <= blockNumber {
       let logger = Logging.createChild(~params={"url": serverUrl})
+      let delayMilliseconds = 100
       logger->Logging.childWarn(
-        `Block #${blockNumber->Int.toString} not found in hypersync. HyperSync runs multiple instances of hypersync and it is possible that they drift independently slightly from the head. Retrying query in 100ms.`,
+        `Block #${blockNumber->Int.toString} not found in HyperSync. HyperSync has multiple instances and it's possible that they drift independently slightly from the head. Indexing should continue correctly after retrying the query in ${delayMilliseconds->Int.toString}ms.`,
```

Comment:
I hope the warning will cause fewer worries. Probably we can even turn it to trace.

Reply:
Awesome 👍 
Agreed it doesn't need to be a warning log